src/text_render.rs

```diff
@@ -69,7 +69,7 @@ impl TextRenderer {
         queue: &Queue,
         atlas: &mut TextAtlas,
         screen_resolution: Resolution,
-        text_areas: &[TextArea<'a, 'b>],
+        text_areas: impl Iterator<Item = TextArea<'a, 'b>> + Clone,
```

Comment:
With the changes from #33 merged, could we remove the `Clone` requirement?

Reply:
You are right! It seems we can merge both loops and iterate only once. Done that in 2f7239b22827fca4ee03666094972ed8d6f9ac7d.